data/human/campaign events.txt

```diff
@@ -1789,7 +1789,7 @@ event "liberation of Poisonwood"
 		fleet "Human Miners" 2000
 	planet "Poisonwood"
 		spaceport `The starport is a hodgepodge of small metal structures, built from the remains of Navy and pirate ships that fought for control of this world. The locals here welcome you to their planet heartily.`
-		spaceport `	The tavern in the center of the port is named The Parting Glass. Its walls are covered with photographs from after the war: liberated slaves boarding Navy ships to leave this planet that had brought them so much pain; Navy officers leaving their ships to retire on this world that they liberated; friends gone away; friends left behind. There are newer pictures, too: a Cruiser and two Falcons, side by side; armored troops walking away from a factory; a familiar captain talking to a man you know as Commander Nguyen. It is a strange feeling to be shown among heroes of a history that you have had only the smallest effect on.`
+		spaceport `	The tavern in the center of the port is named The Parting Glass. Its walls are covered in photographs from after the war, all taken here on Poisonwood: freed slaves boarding Navy ships to leave the planet that brought them so much pain; Navy officers disembarking to retire on the world that they liberated; friends gone away; friends left behind. There are newer pictures, too: a Cruiser and two Falcons, side by side; armored troops walking away from a factory; a familiar captain talking to a man you know as Commander Nguyen. It is a strange feeling to be shown among heroes of a history that you've had only the smallest effect on.`
```

Comment:
This paragraph is based off of the pre-war Poisonwood spaceport description. If you're changing the wording of this description, then I don't see any reason why you wouldn't also change that.
https://github.com/endless-sky/endless-sky/blob/c591b86cda71d35ee1b2fb08104ea462ca801d76/data/map%20planets.txt#L3953

Reply:
I didn't notice this description when I made the edit. My bad. It's updated now.